### DISCUSSION TITLE
[Ente Auth] Keep Icon upon changing the entry title / also allow upload of custom icons

### DISCUSSION THREAD

**Discussion by alexander-rokov** [ente-io/ente#3115] [Enhancements]
When creating a new entry, e.g. Google Mail Account, the correct Google icon is shown, but as soon as I edit the title to e.g. Google - private email, the Google icon is replaced by a generic icon. It would be much appreciated if the entry keep the Google icon. Google is just an example, the app (mobile and desktop) does this for all type of services. It would also be very much appreciated, if we could select/upload a custom icon for an entry.

**Comment by martinstoeckli** [comment]
Is this difficult to solve because the url is not stored and only available when creating the new entry?

**Comment by ua741** [comment]
In 4.2.0, you can manually select the icon from the list of icons.

**Reply by martinstoeckli** [reply]
Nice, works well for me.
